CMakeLists.txt

```diff
@@ -33,19 +33,20 @@ if (HYPER_TOOLCHAIN)
 endif ()
 
 if (NOT HYPER_TOOLCHAIN)
-    set(HYPER_TOOLCHAIN "gcc")
-elseif (NOT HYPER_TOOLCHAIN STREQUAL "gcc" AND
-        NOT HYPER_TOOLCHAIN STREQUAL "clang")
-    message(FATAL_ERROR "Unknown toolchain '${HYPER_TOOLCHAIN}'")
-endif ()
-
-if ($ENV{CLION_IDE})
```

Comment:
I don't necessarily disagree with moving clion detection under that if statement, but that seems like it belongs in a separate commit?

Something like: `don't attempt to detect clion if a toolchain is explicitly specified` (or better wording 🙂 )

Reply:
Moved to "build: disable CLion workaround if toolchain is explicitly specified" cbab01b